lms/services/lti_user.py

```diff
@@ -37,6 +38,7 @@ def from_lti_params(self, application_instance, lti_params) -> LTIUser:
             lti={
                 "course_id": lti_params.get("context_id"),
                 "assignment_id": lti_params.get("resource_link_id"),
+                "product_family": Family.from_launch(lti_params),
```

Comment:
While building a LTIUser for the first time (ie, from a launch, not one that we serelized before) we get the product from the LTI params.